File: python_ToSXs/python/05-uocsoboiso/102.py
arrInt = [14, 12, 3, 5, 6]
number = 3

result = "Không tồn tại"

for elm in arrInt:
    if elm != 0 and number % elm == 0:
        result = "Tồn tại"
        break

print(result)

File: cpp_e9zMZ/cpp/05-uocso-boiso/102.cpp
#include <iostream>
using namespace std;

int main() {
    int arrInt[] = {14, 12, 3, 5, 6};
    int number = 9;
    string result = "Không tồn tại";
    bool flagCheckExist = false;

    for (int elm : arrInt) {
        if (elm != 0 && number % elm == 0) {
            flagCheckExist = true;
            break;
        }
    }

    if (flagCheckExist == true) {
        result = "Tồn tại";
    }

    cout << result;
    return 0;
}